include/alpaka/kernel/TaskKernelGenericSycl.hpp

```diff
@@ -44,7 +42,7 @@
                 sycl::nd_item<TDim::value> work_item) [[intel::reqd_sub_group_size(sub_group_size)]]                  \
             {                                                                                                         \
                 auto acc = TAcc{item_elements, work_item, dyn_shared_accessor, st_shared_accessor};                   \
-                core::apply(                                                                                          \
+                std::apply(                                                                                           \
```

Comment:
Side note: These white spaces behind the opening `(` are added by clang-format. Removing them makes the CI job complain.